src/Notification/NotificationServiceProvider.php

```diff
@@ -30,34 +30,43 @@ public function boot()
      */
     public function registerNotificationTypes()
     {
-        $blueprints = [
-            DiscussionRenamedBlueprint::class => ['alert']
-        ];
+        $this->app->singleton('flarum.notification.blueprints', function () {
+            return [
+                DiscussionRenamedBlueprint::class => ['alert']
+            ];
+        });
+
+        $blueprints = $this->app->make('flarum.notification.blueprints');
```

Comment:
Feels like this (and the logic that it uses) would make more sense to have in boot, after all extensions have done their thing

Reply:
*facepalm*